contracts/crowdsale/validation/TimedCrowdsale.sol

```diff
@@ -11,15 +11,15 @@ import "../Crowdsale.sol";
 contract TimedCrowdsale is Crowdsale {
   using SafeMath for uint256;
 
-  uint256 public openingTime;
-  uint256 public closingTime;
+  uint256 private openingTime_;
+  uint256 private closingTime_;
 
   /**
    * @dev Reverts if not in crowdsale time range.
    */
   modifier onlyWhileOpen {
     // solium-disable-next-line security/no-block-members
```

Comment:
We can remove this Solium annotation because we moved the expression to a function. (Good idea btw!)